src/cluster_legacy.c

```diff
@@ -903,8 +903,18 @@ int clusterSaveConfig(int do_fsync) {
 
 void clusterSaveConfigOrDie(int do_fsync) {
     if (clusterSaveConfig(do_fsync) == C_ERR) {
-        serverLog(LL_WARNING, "Fatal: can't update cluster config file.");
-        exit(1);
+        if (!server.cluster_ignore_disk_write_error) {
+            serverLog(LL_WARNING, "Fatal: can't update cluster config file");
+            exit(1);
+        } else {
+            static mstime_t last_log_time_ms = 0;
+            const mstime_t log_interval_ms = 10000;
```

Comment:
We may need a configuration item to control this in the future, currently this logic is copy from `replica-ignore-disk-write-errors`